allennlp/service/server_sanic.py

```diff
@@ -8,34 +8,36 @@ def run(port: int) -> None:
     print("Starting a sanic server on port {}.".format(port))
     app = make_app()
     # TODO(joelgrus): make this configurable
-    app.servables = ServableCollection.default()
+    app.predictors = PredictorCollection.default()
```

Comment:
I'd rather have that logic here, but I could change it to a function that returns a `Dict[str, Predictor]` rather than a class, how about that?